app/javascript/src/ActiveDocs/customize/wrappedComponents/Parameters.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import 'ActiveDocs/customize/customizeOAS.scss'
+
+const findParamInService = (paramName, service) => {
+  return service[paramName]
+}
+
+const Select = ({value, className}) => {
+  return (
+    <div className={`autocomplete-select ${className}`}>
+      <select>
+        <option value={value}>{value}</option>
+      </select>
+    </div>
+  )
+}
+
+export const ParameterRow = (Original, system) => props => {
+  const paramList = props.rawParam._list._tail.array
```

Comment:
hmm, there's no other way to send the param list? this is the way swagger pass it?